### PR TITLE
Introduce booster-ui and implement v9 template management protocol

### PR DESCRIPTION
**Pull request by wankai123**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

part of https://github.com/apache/skywalking/issues/8477

UI template initializes from local template file would implement later, after new template file is ready.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ Release Notes.
 * Introduce the entity of Process type.
 * Set the length of event#parameters to 2000.
 * Limit the length of Event#parameters.
+* Introduce booster-ui and implement template management protocol.
```
Comment (kezhenxu94) [inline CHANGES.md:85]: Maybe break this into 2 lines, one in OAP section (ui template api) and replace the UI in UI section ?

Comment (wu-sheng) [reply CHANGES.md]: I just resolved the conflict, which should have addressed this.

**Comment by wu-sheng** [comment]
I have sent out the warning mail as this breaking the UI totally.

https://lists.apache.org/thread/0r3rnz0vnywdl662clzcbz9gnkc9ztm9

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/template/UITemplate.java [hunk 2]
```diff
@@ -37,54 +37,48 @@
...
-    public static final String ACTIVATED = "activated";
+    public static final String UPDATE_TIME = "update_time";
     public static final String DISABLED = "disabled";
 
-    @Column(columnName = NAME)
-    private String name;
-    @Column(columnName = TYPE, storageOnly = true)
-    private String type;
+    @Column(columnName = TEMPLATE_ID)
+    private String templateId;
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/template/UITemplate.java:50]: If this has been `ID`, we may not need this column?

**Review by wu-sheng — APPROVED** [review]

File: CHANGES.md [hunk 3]
```diff
@@ -85,12 +85,14 @@ Release Notes.
 * Support large service/instance/networkAddressAlias list query by using ElasticSearch scrolling API, add `metadataQueryBatchSize` to configure scrolling page size.
 * Change default value of `metadataQueryMaxSize` from `5000` to `10000`
 * Replace deprecated Armeria API `BasicToken.of` with `AuthToken.ofBasic`.
+* Implement v9 UI template management protocol.
```
Comment (kezhenxu94) [inline CHANGES.md:88]: A reminder, we should overhaul https://github.com/apache/skywalking/tree/master/dist-material/release-docs/licenses/ui-licenses accordingly as the UI project is completely replaced and must have some dependencies' changes

Comment (wu-sheng) [reply CHANGES.md:88]: Yes, we will.

Comment (wu-sheng) [reply CHANGES.md:88]: FYI @Fine0830

Comment (kezhenxu94) [reply CHANGES.md:88]: Created https://github.com/apache/skywalking/issues/8638 to track this task.

**Review by kezhenxu94 — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**